arangod/V8Server/v8-analyzers.cpp

```diff
@@ -443,6 +444,25 @@ void JS_Get(v8::FunctionCallbackInfo<v8::Value> const& args) {
   // end of parameter parsing
   // ...........................................................................
 
+  const auto analyzerVocbase = arangodb::iresearch::IResearchAnalyzerFeature::extractVocbaseName(name);
+  if(!analyzerVocbase.empty() && vocbase.name() != analyzerVocbase   && 
+    arangodb::StaticStrings::SystemDatabase != analyzerVocbase) {
+    std::string errorMessage("Analyzer '");
+    errorMessage.append(name)
+      .append("' is not accessible. Only analyzers from current database ('")
+      .append(vocbase.name())
+      .append("')");
+    if (vocbase.name() != arangodb::StaticStrings::SystemDatabase) {
+      errorMessage.append(" or system database");
+    }
+    errorMessage.append(" are available");
+    TRI_V8_THROW_EXCEPTION_MESSAGE( 
+      TRI_ERROR_FORBIDDEN, // code
+      errorMessage
+    );
+    return;
```

Comment:
extra `return` statement is not necessary here, as the macro `TRI_V8_THROW_EXCEPTION_MESSAGE` always adds a `return` statement.